index.js

```diff
@@ -81,15 +81,13 @@ module.exports = ({ types: t }) => {
               }
             }
             const paramName = factory && factory.params[i];
-            requireExpressions.push(createRequireExpression(el, paramName));
+            newNodes.push(createRequireExpression(el, paramName));
           });
         }
 
         if(factory) {
           const factoryArity = factory.params.length;
-          let replacementFuncExpr = t.functionExpression(null, [], t.blockStatement(
-            requireExpressions.concat(factory.body.body)
-          ));
+          let replacementFuncExpr = t.functionExpression(null, [], t.blockStatement(factory.body.body));
```

Comment:
We need to remove the arguments, so we still need the create a new function expression. However, upon closer inspection, we don't need to create a new block statement. Can reuse the factory function's existing one.